src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,54 @@
+package duke.parser;
+
+import duke.command.*;
+import duke.exception.KoraException;
+
+/**
+ * Parser class process user input and generates command.
+ */
+public class Parser {
+
+    /**
+     * Class constructor of Parser.
+     */
+    public Parser() {
+    }
+
+    /**
+     * Processes user input and returns a command based on data processed.
+     * @param userInput String input from user.
+     * @return Command to be executed.
+     * @throws KoraException For invalid input.
+     */
+    public static Command parse(String userInput) throws KoraException {
+        String line = "------------------------------" + "\n";
```

Comment:
I cannot see where this variable is used... In addition, do you think we should make this "line" consistent with the "line" you declared in the Command.java? I personally think it is a good practice to make them refer to a single variable. In this way, you only need to modify at one place to change your display instead of modifying every occurrence :) .

Reply:
Okay thanks! I think that will be better :)